cg/models/devices/illumina_run_directory_data.py

```diff
@@ -36,53 +36,53 @@
 }
 
 
-class FlowCellDirectoryData:
-    """Class to collect information about flow cell directories and their particular files."""
+class IlluminaRunDirectoryData:
+    """Class to collect information about sequencing run directories and their particular files."""
 
-    def __init__(self, flow_cell_path: Path):
-        LOG.debug(f"Instantiating FlowCellDirectoryData with path {flow_cell_path}")
-        self.path: Path = flow_cell_path
+    def __init__(self, sequencing_run_path: Path):
+        LOG.debug(f"Instantiating FlowCellDirectoryData with path {sequencing_run_path}")
+        self.path: Path = sequencing_run_path
         self.machine_name: str = EMPTY_STRING
         self._run_parameters: RunParameters | None = None
         self.run_date: datetime.datetime = datetime.datetime.now()
         self.machine_number: int = 0
-        self.base_name: str = EMPTY_STRING  # Base name is flow cell-id + flow cell position
+        self.base_name: str = EMPTY_STRING  # Base name is sequencing run-id + flow cell position
```

Comment:
Would it not make sense to call it flow cell id here? Since we are in the IlluminaRunDirectoryData?

Reply:
Indeed, I think the comment should say flow cell-id as well.